app/lib/account/models.dart

```diff
@@ -21,6 +21,11 @@ class User extends db.ExpandoModel {
 
   @db.DateTimeProperty()
   DateTime created;
```

Comment:
If this and `oauthUserId` can suddenly be `null` we should document this in documentation comments. Otherwise, it's hard to know what the state of these properties is.

Example:
```dart
/// [DateTime] the [User] entity was created.
///
/// This may be `null` if the user [isDeleted] and the [User] entity is retained for audit purposes.
@db.DateTimeProperty()
DateTime created;
```